css/obBase.css

```diff
@@ -76,10 +76,24 @@ textarea {
   font-weight: 500;
 }
 
+input[type="file"] {
+    padding-left:10px;
+    padding-top:15px;
+    color: #fff;
+}
+
+input[type="checkbox"] {
+    margin-left:15px;
+    margin-top:20px;
+}
+
```

Comment:
Remove this.

The style you need already exists, just add the fieldItem class to the checkbox element.